### ISSUE TITLE
Проблема со сборкой под ios новой версии 0.9.0.

### ISSUE THREAD

**Issue by svchernyshov** [vtr0n/react-native-yandex-mobile-ads#23]
Получаю следующую ошибку при сборке:

![photo_2024-02-08_16-29-41](https://github.com/vtr0n/react-native-yandex-mobile-ads/assets/22872245/e5d8e37b-2de7-49ef-958c-d5cd3c24bb1c)

Подскажите, что не так?

**Comment by vtr0n** [comment]
Постараюсь на выходных глянуть

**Comment by svchernyshov** [comment]
> Постараюсь на выходных глянуть

Удалось ли посмотреть? Есть идеи о возможных причинах этой ошибки?

**Comment by svchernyshov** [comment]
Если я обновляю cocoapods, то вижу другую ошибку при сборке под ios.
![Uploading image.png…]()

**Comment by renatomserra** [comment]
Facing the same issue on iOS

**Comment by vtr0n** [comment]
Я не заметил, что надо менять интерфейсы и мигрировать на [новую версию](https://yandex.ru/support2/mobile-ads/ru/dev/ios/release/6-0-0-migration).
Сейчас вообще нету времени на это:( Там не много, но нужно разобраться
Пока что роллбекнул ios на старую версию

**Comment by renatomserra** [comment]
Used the latest page and seems fine. thank you vtr0n!
One thing is i had to use `demo-banner-yandex` test id, the one on example isnt valid, dont know if its legacy or just a placeholder

**Comment by svchernyshov** [comment]
Спасибо за fix!:)
Но все равно, очень ждем, когда обновите версию sdk под ios.:)

**Comment by AntonovAD** [comment]
> Я не заметил, что надо менять интерфейсы и мигрировать на [новую версию](https://yandex.ru/support2/mobile-ads/ru/dev/ios/release/6-0-0-migration). Сейчас вообще нету времени на это:( Там не много, но нужно разобраться Пока что роллбекнул ios на старую версию

Посмотрите, пожалуйста, как будет время 🙏🏻

**Comment by trickyc0d3r** [comment]
Да, там из-за старой версии - на новой архитектуре реакт нейтив не собирается билд в иос. Много дублированных символов.

**Comment by AntonovAD** [comment]
Пожалуйста, подскажите, у вас появилось время на миграцию в ios?

**Comment by codeitnos** [comment]
> Я не заметил, что надо менять интерфейсы и мигрировать на [новую версию](https://yandex.ru/support2/mobile-ads/ru/dev/ios/release/6-0-0-migration). Сейчас вообще нету времени на это:( Там не много, но нужно разобраться Пока что роллбекнул ios на старую версию

Здравствуйте. Спасибо за вашу библиотеку! Очень хочется, что бы вы обновили её на новую версию SDK яндекса! Спасибо!

**Comment by AndoInvest** [comment]
Спешу обрадовать

РСЯ начала работу в направлении разработки официальной библиотеки для React Native.  Сейчас им нужно побольше информации о нуждах разработчиков приложений, поэтому прошу всех, кто заинтересован, пройти опрос (всего 5 легких вопросов).

Пожалуйста, оставьте свои комментарии в коротком опросе - https://forms.yandex.ru/surveys/13648758.46cebf0f701fe31df85ffcc87f323722134d2630/

Если нужны подробности можете связаться с менеджером по работе с партнерами
alexandervas@yandex-team.com
